plugin/src/Types.lua

```diff
@@ -25,6 +25,7 @@ local ApiInstanceUpdate = t.interface({
 	id = RbxId,
 	changedName = t.optional(t.string),
 	changedClassName = t.optional(t.string),
+	requiresRecreate = t.optional(t.boolean),
```

Comment:
I don't think we should update this interface at all. The server generates most of the patches that the Rojo plugin handles and it does not populate `requiresRecreate`. We can calculate this when we go to apply a patch instead.